server/routes/activities/manage-allocations/handlers/deallocationReason.test.ts

```diff
@@ -85,4 +88,17 @@ describe('Route Handlers - Deallocation reason', () => {
       expect(res.redirect).toHaveBeenCalledWith('check-answers')
     })
   })
+
+  describe('validation', () => {
```

Comment:
There was no test coverage for this.

Reply:
Could add a test for validation passing too?

Reply:
Add passing case too.